docs/resources/Channel.md

```diff
@@ -338,12 +338,16 @@ Represents a message sent in a channel within Discord.
 | Field            | Type      | Description                                                                                   |
 | ---------------- | --------- | --------------------------------------------------------------------------------------------- |
 | id               | snowflake | id of the sticker                                                                             |
-| pack_id          | snowflake | id of the pack the sticker is from                                                            |
+| pack_id?         | snowflake | id of the pack the sticker is from                                                            |
 | name             | string    | name of the sticker                                                                           |
 | description      | string    | description of the sticker                                                                    |
-| tags?            | string    | a comma-separated list of tags for the sticker                                                |
-| asset\*          | string    | sticker asset hash                                                                            |
+| tags             | string    | a unicode emoji representing the sticker's expression                                         |
```

Comment:
`tags` is still a comma-separated list for Nitro stickers though?

Reply:
You're right, thanks for noting, updated for clarity!